builtin/audit/file/backend_test.go

```diff
@@ -0,0 +1,78 @@
+package file
+
+import (
+	"os"
+	"strconv"
+	"testing"
+
+	"github.com/hashicorp/vault/audit"
+	"github.com/hashicorp/vault/helper/salt"
+)
+
+func TestAuditFile_fileModeNew(t *testing.T) {
+	salter, _ := salt.NewSalt(nil, nil)
+	
+	modeStr := "0777"
+	mode, err := strconv.ParseUint(modeStr, 8, 32)
+	
+	file := "auditTest.txt"
+	
+	config := map[string]string{
+		"path": file,
+		"mode": modeStr,
+	}

```

Comment:
`test-fixtures` are meant for files that need to exist for testing, not files being created; this should be a temp file.